shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -436,6 +438,20 @@ - (nullable id)accessibilityElementAtIndex:(NSInteger)index {
     return _semanticsObject;
   }
   SemanticsObject* child = [_semanticsObject children][index - 1];
+
+  // This 'if' block handles adding accessibility support for the embedded platform view.
+  // We first check if the child is a semantic node for a platform view.
+  // If so, we add the platform view as accessibilityElements of the child.
```

Comment:
Avoid documenting the how, and instead focus on the what/why. Maybe something like:
`// If the child is a semantic node for a platform view, inject the platform view into the iOS accessibility tree.`

Reply:
done